### PR TITLE
docs: add template authoring guidelines to readme

### PR DESCRIPTION
**Pull request by anfibiacreativa**
As discussed with @spboyer, this PR adds reference to template artifacts for standardization for discoverability.

### CODE REVIEW

**Comment by anfibiacreativa** [comment]
@spboyer The wiki page does not have a PR process. I will wait for comments/change requests or approval to this one, to edit the wiki.

**Review by vhvb1989 — APPROVED** [review]

**Pull request merged**
